src/ui/public/index_patterns/_index_pattern.js

```diff
@@ -95,9 +95,21 @@ export function IndexPatternProvider(Private, Notifier, config, kbnIndex, Promis
     return promise;
   }
 
-  function containsFieldCapabilities(fields) {
-    return _.any(fields, (field) => {
-      return _.has(field, 'aggregatable') && _.has(field, 'searchable');
+  function isFieldRefreshRequired(indexPattern) {
+    if (!indexPattern.fields) {
+      return true;
+    }
+
+    return indexPattern.fields.every(field => {
+      // added in 5.0 #8421
```

Comment:
++ I agree. Enough to link to the PR, e.g. "Added in #x"? (So we don't have the diff between "added in" and "slotted for"). The PR will have the details about version and why it's added etc.